integration-tests/tests/common/mod.rs

```diff
@@ -808,9 +810,7 @@ impl<'a> TestServerConfigurator<'a> {
                     &self.test_context.runner_path(&self.server_number),
                 );
 
-                let output = Command::new("whoami").output().unwrap();
```

Comment:
We originally had a dependency for getting the current user, and changed it to calling the `whoami` command.

I'm reluctant to adding back another dependency (with adds many other dependencies on cascade) when we can get this info without any external crate

Reply:
Should I remove the `systeminfo` dependency then and use `whoami` in all cases?. It doesn't seem the more "elegant" way to do this but I understand your point...

Reply:
https://github.com/fdo-rs/fido-device-onboard-rs/commit/574d4e4675af3f57ba657df6ed17578e1f5305ea

:arrow_up: change from `users` crate to `whoami`

`whoami` is part of coreutils, which seems pretty safe to rely on for integration tests.

Reply:
It looks like `whoami` has been deprecated in some distributions in favor of `id` (also in coreutils package) should I use that command instead?

Reply:
Sure; ‘id -un` operates the same as `whoami` according to the manpage